spec/features/user_can_adjust_quantity_of_item_in_cart_spec.rb

```diff
@@ -12,12 +12,12 @@
       end
       click_link "View Cart"
       within "#item_row_#{item_one.id}" do
-        fill_in 'Quantity', with: "3"
+        fill_in 'quantity-field', with: "3"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.